lib/darwinning/evolution_types/mutation.rb

```diff
@@ -21,7 +21,7 @@ def pairwise?
       def mutate(members)
         members.map do |member|
           if rand < mutation_rate
-            re_express_random_genotype(member)
+            send(@mutation_method, member)
```

Comment:
What do you think of using the same pattern as mutation_rate and using an attr_reader?